src/parser.R

```diff
@@ -9,137 +9,149 @@ library(here) # For relative working directory. Looks for where there is a file
 #
 # this function looks at a multiple files, breaks it down by month, day, 
 # parses it and then binds everything together to form one dataset
-crawlerResultPath <- file.path(getwd(),                       # gets the working root directory  
-                               'crawler_result_Conversion',   # folder where we want to save the PDFs and 
-                               fsep = .Platform$file.sep)     # makes sure the path is platform agnostic
-
-
-# List of RegEx used
-listOfTxtFiles = list.files(crawlerResultPath, pattern = ".txt")
-dayStartRegEx <- "^For Date:\\s*(\\d{2}\\/\\d{2}\\/\\d{4})\\s*-\\s*(\\w*)"
-callRegEx <- "^(\\s|\\d{2}-\\d*)\\s*(\\d{4})\\s*(.*?)\\s{2,}(.*)"
-locationRegEx <- "^\\s*Location/Address:\\s*(.*)"
-officeRegEx <- "^\\s*ID:\\s*(.*)"
-callTakerRegEx <- "^\\s*Call Taker:\\s*(.*)"
-dispClrdRegEx <- "^\\s*Disp-(\\d{2}:\\d{2}:\\d{2})\\s*(Arvd-\\d{2}:\\d{2}:\\d{2})?\\s*Clrd-(\\d{2}:\\d{2}:\\d{2})"
-wasArrestMadeRegEx <- "^\\s*Refer To Arrest:\\s*(.*)"
-referToSummonRegEx <- "^\\s*Refer To Summons:\\s*(.*)$"
-summonRegEx <- "^\\s*Summons:\\s*(.*)$"
-arrestRegEx <- "^\\s*Arrest:\\s*(.*)$"
-addressRegEx <- "^\\s*Address:\\s*(.*)$"
-ageRegEx <- "^\\s*Arrest:\\s*(.*)$"
-chargesRegEx <- "^\\s*Charges:\\s*(.*)$"
-
-parseTxtToDf <- function(filePath){
+
+crawlerResultPath <- "./crawler_result_Conversion" # text and pdf location
```

Comment:
So is the understanding that this code is ran from the `src` directory? If that's the case, are you moving the `crawler_result_Conversion` path to be a subdirectory of the `src` path?

Reply:
I still think you should be explicit with the relative path to the`src` directory. This could cause some headache in the future. Consistency will avoid future troubles, especially with the travis testings.